kubejs/assets/kubejs/lang/pl_pl.json

```diff
@@ -0,0 +1,11 @@
+{
+  ///// FLUIDS NAME HERE /////
+  "fluid.kubejs.molten_calorite" : "Kaloryt",
+  "fluid.kubejs.molten_ostrum" : "Ostrum",
+  "fluid.kubejs.molten_desh" : "Desh",
+  "fluid.kubejs.plasma_fluid" : "Płyn Plazmy Kwarkowo-Gluonowej",
+  "fluid.kubejs.hellfire" : "Piekielnyogień",
```

Comment:
"Piekielny ogień".
Why was a space omitted here?

Reply:
it was omitted in the original as well

Reply:
Though in Polish, usually spaces aren't omitted.

Imagine that you are seeing "sklepzaplikacjami" instead of "sklep z aplikacjami".